test/es6/bug_OS14880030.js

```diff
@@ -0,0 +1,59 @@
+//-------------------------------------------------------------------------------------------------------
```

Comment:
nit: instead of new file does it make sense to add these tests under test\bugs\misc_bugs.js (and make use of unittestframework) ?

Reply:
Agree it would be nicer to use unittestframework but I want to control deferparse flag to this test so I want to keep it separate from misc_bugs.js

Reply:
Refactored to use the unittestframework 👍